openbb_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -213,7 +213,6 @@ def __init__(
                 "--currency": None,
                 "-c": "--currency",
                 "--raw": {},
-                "--show-all": {},
```

Comment:
I'm actually not sure whether this should be removed or not. Based on [this comment](https://github.com/OpenBB-finance/OpenBBTerminal/issues/3249#issuecomment-1299038630) it kinda seems like it should?

Appreciate any clarifications.

Reply:
This should not be here. I added it then took it away from argparse but not autocomplete